### DISCUSSION TITLE
It should be possible to set a color in the FluentTab

### DISCUSSION THREAD

**Discussion by kephax** [microsoft/fluentui-blazor#2412] [Ideas]
Currently you cannot change the color of the Icon and Title of the FluentTab component

**Comment by vnbaaij** [comment]
That is definitely possible. There is an example of that in the demo site: https://www.fluentui-blazor.net/Tabs#customheaderandpanel

**Comment by kephax** [comment]
According to the example we need to do it this way:

`<FluentTab Id="tab-3">
        <Header>
            <FluentIcon Value="@(new Icons.Regular.Size16.LeafThree())" Color="@Color.Error" />
            <span style="color:forestgreen;">February</span>
        </Header>
        <Content>
            Content 2
        </Content>
    </FluentTab>`


It would be more consistent if we could do something like this:

`<FluentTab IconColor="Color.Custom" CustomColor="#FFE699" />`

**Reply by vnbaaij** [reply]
No, we won't do it that way because it is not following the Fluent design then. 
If you really want to do it, you can use the span but we won't make it easier to deviate

**Reply by kephax** [reply]
Thanks for the reply. It would be nice if we can manually change the color of the line under a tab, currently only the main accent color is used.
There can be scenarios where we want to force a color on an icon and title (like in the example above) but also in the line. 
Is there any plan to allow this?

**Reply by vnbaaij** [reply]
This is again something which is coming from the Fluent design and is not something we will be want to make available for easy changing.

The only way to change that color is through CSS. The web component we wrap also does not expose a way to change that.
